pkgs/development/libraries/gnutls/default.nix

```diff
@@ -98,6 +100,9 @@ stdenv.mkDerivation rec {
       "--with-unbound-root-key-file=${dns-root-data}/root.key"
       (lib.withFeature withP11-kit "p11-kit")
       (lib.enableFeature cxxBindings "cxx")
+    ] ++ lib.optionals (stdenv.hostPlatform.isMinGW) [
+      "--disable-non-suiteb-curves"
```

Comment:
I feel that `--disable-non-suiteb-curves ` could use a comment. (Is it really mingw specific, or is it just to avoid mass-rebuilds?) I looked at the source briefly and it only says it disables "Suite B" curves -- but I don't know what that is. 

However, the build also succeeds without this configure flag. So what do we need it for?